cmd/swarm/main.go

```diff
@@ -154,6 +154,26 @@ var (
 		Usage:  "Domain on which to send Access-Control-Allow-Origin header (multiple domains can be supplied separated by a ',')",
 		EnvVar: SWARM_ENV_CORS,
 	}
+	SwarmStoreDbPath = cli.StringFlag{
+		Name:   "dbpath",
+		Usage:  "Path to leveldb chunk DB (default <$GETH_ENV_DIR>/swarm/bzz-<$BZZ_KEY>/chunks)",
+		EnvVar: SWARM_ENV_STORE_DB_PATH,
+	}
+	SwarmStoreDbCapacity = cli.IntFlag{
+		Name:   "dbcapacity",
+		Usage:  "Number of chunks (5M is roughly 20-25GB) (default 5000000)",
+		EnvVar: SWARM_ENV_STORE_DB_CAPACITY,
+	}
+	SwarmStoreCacheCapacity = cli.IntFlag{
+		Name:   "cachecapacity",
+		Usage:  "Number of recent chunks cached in memory (default 5000)",
+		EnvVar: SWARM_ENV_STORE_CACHE_CAPACITY,
+	}
+	SwarmStoreRadius = cli.IntFlag{
+		Name:   "radius",
```

Comment:
I think the flags should be prefixed with `store.`, for example:

```
--store.path
--store.size
--store.cache.size
--store.radius
```

Reply:
👍